web/blog/2022-05-31-filip-intro.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Meet the team - Filip Sodić, Founding Engineer
+authors: [matijasos]
+image: /img/shayne-intro-banner.png
+tags: [wasp, new-hire]
+---
+
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import InBlogCta from './components/InBlogCta';
+import WaspIntro from './_wasp-intro.md';
+
+![Welcome Filip!](../static/img/shayne-intro-banner.png)
+
+Find Filip on [GitHub](https://github.com/sodic).
+
+<!--truncate-->
+
+<WaspIntro />
+<InBlogCta />
+
+#### Let’s start with two truths and a lie about yourself.
+
+- I found out about Wasp through a friend/former colleague and still haven’t
+  bought him a drink to say thanks.
+- I’ve created my own Turing-complete functional programming language that
+  compiles to JavaScript.
+- I wasted nearly 30 hours customizing my setup to maximize productivity and
+  probably ended up in a less productive state
+
+#### Which statement was the lie above? Any interesting stories to share?
+
+The last one was a lie. I wish it were only 30 hours :)
+
+I did create my own language, but it’s still far from being practical. For one,
+it can’t print anything to the screen (or perform any side effects).
```

Comment:
Where is link to your language, to github repo. We want link, we want link

Reply:
Another rickroll or the real thing?